src/System.Data.SqlClient/src/System/Data/ProviderBase/DbConnectionPool.cs

```diff
@@ -719,6 +720,12 @@ private DbConnectionInternal CreateObject(DbConnection owningObject, DbConnectio
                 {
                     throw;
                 }
+#if netcoreapp
+                if (!IsBlockingPeriodEnabled())
```

Comment:
I am OK with this approach. However an alternative could have been to call a function here `CheckPoolBlockingPeriod()`;
There there could have been a partial DbConectionPool for netstandard which leaves the function empty and another DbConnectionPool.netcoreapp.cs compiled when netcoreapp DLL is built, which implements 
```C#
internal void CheckPoolBlockingPeriod() 
{ 
      if (!IsBlockingPeriodEnabled()) 
     {
               throw;
     } 

}

```